src/components/containers/signed-token-refresh.tsx

```diff
@@ -14,8 +14,8 @@ import Config from "../../config";
  * Component props
  */
 interface Props {
-  adminToken?: AccessToken;
-  onAdminLogin: (keycloak: KeycloakInstance, adminToken: AccessToken) => void;
+  signedToken?: AccessToken;
+  onSignedLogin: (keycloak: KeycloakInstance, signedToken: AccessToken) => void;
```

Comment:
Can be simplified like this:
```suggestion
  onSignedLogin: typeof signedLogin;
```